### ISSUE TITLE
Invalid tracer argument on debug_traceTransaction

### ISSUE THREAD

**Issue by mpauls2839** [EthTx/ethtx#190]
I'm seeing the debug_traceTransaction method consistently fail across different node providers. Specifically, I belive it may be related to the prvided tracer argument. 
`{
  "jsonrpc": "2.0",
  "id": 2,
  "error": {
    "code": -32602,
    "message": "invalid 2nd argument: options 'tracer': invalid tracer value"
  }
}`

**Comment by kchojn** [comment]
Hey @mpauls2839,
Is this related to `ethtx`? If so, can you give more details? Transaction hash, the provider you are using, etc?
Regards

**Comment by mpauls2839** [comment]
Yes, I believe this is failing on the ethtx call to debug_traceTransaction. I've tried both quicknode and Alchemy.com as my node providers. It seems to fail on any transaction, but here's a sample one: 0xe8cc27de3feb65a0403d2f2f772e74db2e8ded12f5f6343b75005894b3820817

Here's a screenshot of the call failing. It seems all other calls are successful however.
![image](https://user-images.githubusercontent.com/22549148/216840600-e092b052-c0f6-4247-80ba-1f83fd2868b1.png)

**Comment by mpauls2839** [comment]
Specifically, I believe this error is happening in ethtx/providers/web3_provider.py::line 257

**Comment by kchojn** [comment]
@mpauls2839,
thanks. Are you using free plans from these providers? A full archive node is required for ethtx to work, probably getblock.io for the last 64 blocks has this functionality (for free), otherwise, you need a higher plan.
Let me know and I will check it more deeply 🙌
